test/schema/suggested_genes.js

```diff
@@ -0,0 +1,30 @@
+import { runQuery } from "test/utils"
+const fetch = jest.mock("../../lib/apis/fetch", () => jest.fn())
+fetch.mockReturnValue([
```

Comment:
This test currently fails and I believe it's because I'm not properly mocking out the `fetch` function. Per the [documentation](http://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockcalls) it seems I should be able to create the mock and specify its return value but `mockReturnValue` doesn't seem to be a function on my mock.

Reply:
For the record, the reason this wasn't working is that jest isn't used in this project 🙈  and ended up using sinon and rewire for the test setup